server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -1607,9 +1607,9 @@ public TSStatus createMultiTimeseries(TSCreateMultiTimeseriesReq req) {
 
       boolean isAllSuccessful = true;
 
-      for (Map.Entry<Integer, Boolean> entry : createMultiTimeSeriesPlan.getResults().entrySet()) {
-        if (!entry.getValue()) {
-          isAllSuccessful = false;
+      if (createMultiTimeSeriesPlan.getResults().entrySet().size() > 0) {
+        isAllSuccessful = false;
+        for (Map.Entry<Integer, Exception> entry : createMultiTimeSeriesPlan.getResults().entrySet()) {
           statusList.set(entry.getKey(), RpcUtils.getStatus(TSStatusCode.EXECUTE_STATEMENT_ERROR));
```

Comment:
I think the messages of the exceptions should be put into the statuses.
`RpcUtils.getStatus(TSStatusCode.EXECUTE_STATEMENT_ERROR, entry.getValue().getMessage())`

Reply:
yes, I have fixed it.